lib/zh-cn/diagnosticMessages.generated.json

```
@@ -153,23 +153,23 @@
   "Add_override_modifier_95160": "添加 \"override\" 修饰符",
   "Add_parameter_name_90034": "添加参数名称",
   "Add_qualifier_to_all_unresolved_variables_matching_a_member_name_95037": "将限定符添加到匹配成员名称的所有未解析变量",
-  "Add_this_parameter_95104": "添加“此”参数。",
+  "Add_this_parameter_95104": "添加“this”参数。",
```

```suggestion
  "Add_this_parameter_95104": "添加 \"this\" 参数。",
```

Pointing out for consistency with most other texts